tests/core/cross/cross_test.go

```diff
@@ -57,10 +57,15 @@ var checks = []check{
 
 func TestCross(t *testing.T) {
 	for _, c := range checks {
-		if _, err := os.Stat(*c.file); os.IsNotExist(err) {
-			t.Fatalf("Missing binary %v", *c.file)
+		file := *c.file
```

Comment:
I think you should be able to use `bazel.Runfile(*c.file)` instead of adding this logic (`bazel` is `github.com/bazelbuild/rules_go/go/tools/bazel`).